### DISCUSSION TITLE
[BIOS][Setting] ASUS Interface Control support

### DISCUSSION THREAD

**Discussion by Solarriors** [seerge/g-helper#1857] [Q&A]
Should the BIOS option be disabled with G-helper providing supposedly those functions ?

Will the Turbo boost (CPU+GPU boost) still work with G-helper independent from wattage settings ? I've set mine to 30W PPL1 up to 150W PPL2, Efficient Aggressive at Guaranteed.

The aim is to reduce bloat and improve system resources availability and performance.
Previous read : [What is Armory Crate control support](https://www.reddit.com/r/ASUSROG/comments/s4god8/what_is_armourycrate_control_interface_support_in/)

**Comment by seerge** [comment]
@Solarriors 

> Should the BIOS option be disabled with G-helper providing supposedly those functions ?

If you mean Armoury Crate Interface - it doesn't seem to matter. It's responsible only for an annoying popup asking to install AC itself

> Will the Turbo boost (CPU+GPU boost) still work with G-helper independent from wattage settings ? I've set mine to 30W PPL1 up to 150W PPL2, Efficient Aggressive at Guaranteed.

Turbo boost is just a setting in windows power plan, if windows decides it needs to boost and it has enough power it will do that :) unless you limit power too much

**Comment by Solarriors** [comment]
Thanks for the answer.
But in my UEFI settings I have two options. the aforementioned "ASUS Interface Control support" and the specific "Armory Crate pop-up prompt", so they are different things. However testing has shown not difference in functionality between the former being on or off.

Thanks for the Boost profile explanation. I've found some info and then I stumbled upon the Intel DTT _**Telemetry service**_ , so not the DTT driver itself, which is also uncertain how it affects performance scenarios.

All of these makes me come to my previous remark : Why do IT&Electronics engineers put some "bloatptions" that confuse, have no effects or aren't either the best optimal settings nor "fool-proof".

Thank you again for g-helper.
